src/tokens/bartio.json

```diff
@@ -125,6 +125,13 @@
       "image": "buds.svg",
       "symbol": "BUDS",
       "tags": []
+    },
+    {
+      "address": "0xcC8320B61A6F1719f5FC1c090F26b75dd75cA395",
+      "decimals": 0,
```

Comment:
Blocking this PR until we are able to extensively test 0 decimals. I expect it'll cause problems, at least on the UI.

We'll have to spin up some test API to test it.